wasm/src/lib.rs

```diff
@@ -245,13 +245,13 @@ mod tests {
     }
 
     fn get_test_instruction() -> Instruction {
-        let new_account_id = AccountId::test("mad_hatter", "wonderland");
-        let register_isi = RegisterBox::new(NewAccount::new(new_account_id));
+        let new_account_id = AccountId::new("mad_hatter", "wonderland").expect("Valid");
+        let register_isi = RegisterBox::new(Account::new(new_account_id, []));
```

Comment:
Can we do something with this `[]`? There are too much constructions like that (browser search gives me 14 matches in this diff). There should be a way to construct `Account` just with id

Reply:
yes, this is API preference. I figured that giving signatories is so fundamental that users should be aware that they haven't given any. I'd like others' opinion on this as well

Reply:
I'm resolving this